ViperMonkey Changes June 2018

kirk-sayre-work opened the pull request.

decalage2 commented on vipermonkey/core/__init__.py:
```diff
@@ -139,12 +140,47 @@ def __init__(self):
-                             'workbook_activate', 'auto_close', 'workbook_close']
-
+                             'workbook_activate', 'auto_close', 'workbook_close',
+                             'inkpicture1_painted']
```
is inkpicture1_painted needed here, since _Painted is handled in the callback suffixes below? (and moreover "inkpicture1" is just a default name that can be changed)

kirk-sayre-work commented:
Philippe,

The inkpicture1_painted entry point was initially added to handle some Trickbot maldocs before I added the more general *_painted entry point functionality. I’ve removed the inkpicture1_painted entry point in the latest commit.

Thanks,

Kirk

From: Philippe Lagadec <notifications@github.com>
Sent: Wednesday, August 15, 2018 3:26 PM
To: decalage2/ViperMonkey <ViperMonkey@noreply.github.com>
Cc: Kirk Sayre <Kirk.Sayre@walmart.com>; Author <author@noreply.github.com>
Subject: EXT: Re: [decalage2/ViperMonkey] ViperMonkey Changes June 2018 (#34)


@decalage2 commented on this pull request.

Everything looks good, just one question about inkpicture1_painted.

________________________________

In vipermonkey/core/__init__.py<https://github.com/decalage2/ViperMonkey/pull/34#discussion_r210392954>:

>          # List of entry point functions to emulate.

         self.entry_points = ['autoopen', 'document_open', 'autoclose',

                              'document_close', 'auto_open', 'autoexec',

                              'autoexit', 'document_beforeclose', 'workbook_open',

-                             'workbook_activate', 'auto_close', 'workbook_close']

-

+                             'workbook_activate', 'auto_close', 'workbook_close',

+                             'inkpicture1_painted']

is inkpicture1_painted needed here, since _Painted is handled in the callback suffixes below? (and moreover "inkpicture1" is just a default name that can be changed)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/decalage2/ViperMonkey/pull/34#pullrequestreview-146600091>, or mute the thread<https://github.com/notifications/unsubscribe-auth/Acgz5YkH-sccOVE7Cak06QXCs0Vs7EY_ks5uRIPngaJpZM4UwCTj>.
